panel/widgets/input.py

```diff
@@ -469,6 +469,9 @@ class _DatetimePickerBase(Widget):
     __abstract = True
 
     def __init__(self, **params):
+        if "options" in params:
+            options = list(params.pop("options"))
+            params["enabled_dates"] = options
```

Comment:
I was initially thinking of putting this in the [holoviews.py module where the widget  conversion happens](https://github.com/holoviz/panel/blob/main/panel/pane/holoviews.py#L663), but it felt awkward to add DatetimePicker logic there, so I put it here. Also, since `options` is used for most other widgets, it makes sense to also support options for the DatetimePicker widget too.